src/pages/LandingPage/LandingPage.jsx

```diff
@@ -15,6 +15,16 @@ const LandingPage = () => (
         <h2>It's all in the details</h2>
       </div>
     </div>
+    <div
+        id="about-section"
+        className="w-full flex bg-cover bg-right"
+        style={{ backgroundImage: `url(${about_image})` }}
+    >
+        <div className="p-6 flex self-center">
```

Comment:
Maybe we should have some margin at the bottom moving this text slightly higher? If you look at the text on different browser sizes it sometimes overlaps the car.

Reply:
Resolved